### PR TITLE
Adding step for Engineering Hiring Managers

### PR DESCRIPTION
**Pull request by msennett22**
Adding an additional step for Engineering Hiring Managers that creates greater visibility into final hiring decisions with the broader interviewing team. This will help calibrate for future interviews. 

I think this could make sense as the last item in Step 2 or could live in the Interview Debriefs section below. Open to feedback.

### CODE REVIEW

File: content/talent/resources_for_hiring_managers.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ Please reference the [hiring plan](https://docs.google.com/spreadsheets/d/1Dpf6a
   - [Interview process for each department](interview_process.md)
 - The hiring team (including the Recruiter, Hiring Manager, and all other interviewers) debriefs and makes final hiring decision.
   - This can be done in a synchronous fashion via a debrief Zoom meeting, or asynchronously via Slack.
+    - **Engineering Hiring Managers** are responsible for communicating your hiring decisions to the rest of the interview team to stay informed and calibrate for future hiring decisions.
+      - Post your note in your private Slack channels.
+      - Your note should provide context for why we've decided to hire someone, or why we've decided _not_ to hire someone.
+      - Once it's been posted in Slack, paste your note into Greenhouse so it lives on the candidate page forever.
```
Comment (nicksnyder) [inline content/talent/resources_for_hiring_managers.md]: Suggesting deindenting one stop and minor wording update.

```suggestion
  - **Engineering Hiring Managers** are responsible for communicating your hiring decisions to the rest of the interview team to stay informed and calibrate for future hiring decisions.
    - Post your note in the private Slack channel for the role.
    - Your note should provide context for why we've decided to hire someone, or why we've decided _not_ to hire someone.
    - Once it's been posted in Slack, paste your note into Greenhouse so it lives on the candidate page forever.
```

@devoncoords do we only want to do this on engineering, or should the docs be more general than that?

**Review by nicksnyder — APPROVED** [review]

**Comment by nicksnyder** [comment]
I do think that we should remove the debrief section below and unify it with what you wrote

File: content/talent/resources_for_hiring_managers.md (hunk 1 above)
Comment (devoncoords) [reply content/talent/resources_for_hiring_managers.md]: @nicksnyder let's be more general. Let's change **Engineering Hiring Managers** to **Hiring Managers**

**Review by devoncoords — COMMENTED** [review]
Looks great! Let's just change Engineering Hiring Managers to Hiring Managers to be more general.

**Pull request merged**